include/cpr/session.h

```diff
@@ -45,11 +46,15 @@ class Interceptor;
 class MultiPerform;
 
 class Session : public std::enable_shared_from_this<Session> {
+  private:
+    curl_slist * curlSlist_ = nullptr;
```

Comment:
At the end of this class, there is already a list of private variables. Please move it down there.

Reply:
And please use the following notation:
`curl_slist* resolveCurlList_{nullptr}`;